packages/client/hmi-client/src/workflow/ops/calibrate-ciemss/tera-calibrate-node-ciemss.vue

```diff
@@ -1,190 +1,40 @@
-						<InputNumber id="numIterations" v-model="numIterations"></InputNumber>
-					</div>
-				</div>
-				<div class="flex flex-column gap-2 w-full">
-					<label class="extras-label" for="method">Solver method</label>
-					<Dropdown id="method" :options="ciemssMethodOptions" v-model="method" />
-				</div>
-			</AccordionTab>
-			<!-- <AccordionTab header="Loss"></AccordionTab>
-			<AccordionTab header="Parameters"></AccordionTab>
-			<AccordionTab header="Variables"></AccordionTab> -->
-		</Accordion>
-		<section v-else class="emptyState">
-			<img src="@assets/svg/seed.svg" alt="" draggable="false" />
-			<p class="helpMessage">
-				Connect a model configuration and dataset, then configure in the side panel
-			</p>
-		</section>
-		<Button
-			v-if="modelConfigId && datasetId"
-			class="p-button-sm run-button"
-			label="Run"
-			icon="pi pi-play"
-			@click="runCalibrate"
-			:disabled="disableRunButton"
-		/>
+	<section v-if="!showSpinner" class="emptyState">
+		<img src="@assets/svg/seed.svg" alt="" draggable="false" />
+		<p class="helpMessage">
+			Connect a model configuration and dataset, then configure in the side panel
+		</p>
 	</section>
 	<section v-else>
 		<tera-progress-bar :value="progress.value" :status="progress.status" />
 	</section>
 </template>
 
 <script setup lang="ts">
-import { computed, shallowRef, watch, ref, ComputedRef, onMounted, onUnmounted } from 'vue';
+import { ref, onMounted, onUnmounted } from 'vue';
 import { ProgressState, WorkflowNode } from '@/types/workflow';
-import DataTable from 'primevue/datatable';
-import Button from 'primevue/button';
-import Dropdown from 'primevue/dropdown';
-import Column from 'primevue/column';
-import Accordion from 'primevue/accordion';
-import AccordionTab from 'primevue/accordiontab';
-import InputNumber from 'primevue/inputnumber';
 import {
-	CalibrationRequestCiemss,
-	ClientEvent,
-	ClientEventType,
-	CsvAsset,
-	ModelConfiguration
-} from '@/types/Types';
-import {
-	makeCalibrateJobCiemss,
-	getRunResultCiemss,
 	simulationPollAction,
 	querySimulationInProgress
 } from '@/services/models/simulation-service';
-import { setupModelInput, setupDatasetInput } from '@/services/calibrate-workflow';
-import { ChartConfig, RunResults } from '@/types/SimulateConfig';
-import _ from 'lodash';
 import { Poller, PollerState } from '@/api/api';
-import TeraSimulateChart from '@/workflow/tera-simulate-chart.vue';
 import TeraProgressBar from '@/workflow/tera-progress-bar.vue';
-import { getTimespan } from '@/workflow/util';
 import { logger } from '@/utils/logger';
-import { subscribe, unsubscribe } from '@/services/ClientEventService';
-import {
-	CalibrationOperationCiemss,
-	CalibrationOperationStateCiemss,
-	CalibrateMap
-} from './calibrate-operation';
+import { CalibrationOperationStateCiemss } from './calibrate-operation';
+
+// TODO: FIGURE OUT WHAT TO SHOW IN THIS WORKFLOW NODE WHILE CALIBRATION IS RUNNING FROM THE DRILLDOWN
```

Comment:
Should be safe to remove this. Every single node that can be run will have this same question answered this will not be a unique issue
If it is unique enough to track we should create a github issue rather than leaving this TODO as this todo will be lost to time